src/app/core/authentication/authentication.service.ts

```diff
@@ -94,11 +94,16 @@ export class AuthenticationService {
 
     if (environment.oauth.enabled) {
       let httpParams = new HttpParams();
-      httpParams = httpParams.set('client_id', 'community-app');
+      httpParams = httpParams.set('username', loginContext.username);
+      httpParams = httpParams.set('password', loginContext.password);
+      httpParams = httpParams.set('client_id', `${environment.oauth.appId}`);      
+      httpParams = httpParams.set('client_secret', `${environment.oauth.clientSecret}`);
```

Comment:
If the grant is password, i dont think we need client_secret:
`client_secret= - (optional) - If the application is a “confidential client” (not a mobile or JavaScript app), then the secret is included as well.`

Reply:
Fixed